decidim-budgets/config/locales/es.yml

```diff
@@ -53,7 +53,7 @@ es:
           title: Confirmar voto
         budget_excess:
           close: Cerrar
-          description: Este proyecto sobrepasa el presupuesto máximo y no se puede añadir. Si lo deseas, puede eliminar un proyecto que ya hayas seleccionado o votar con tus preferencias.
+          description: Este proyecto sobrepasa el presupuesto máximo y no se puede añadir. Si lo deseas, puede eliminar un proyecto que ya hayas seleccionado o dar apoyo con tus preferencias.
```

Comment:
http://dle.rae.es/?id=3HswNlq|3HvKC6Q

apoyar
3. tr. Favorecer, patrocinar, ayudar.

In Catalan, it should be `donar suport`:

http://www.diccionari.cat/lexicx.jsp?GECART=0130140
suport
2 Acció d'advocar per alguna cosa (a obtenir, a acordar, etc.), de mantenir-la, de defensar-la.

http://www.diccionari.cat/lexicx.jsp?GECART=0003097
advocar
1 Parlar en favor d'algú o d'alguna cosa

So, semantically, using `donar suport`/`dar apoyo` we are encouraging to vote in favor for something, rather than negatively. Using `votar`/`votar` we are encouraging to give an opinion, but we are not positioning ourselves. I think this is important, but I might be very picky for these things.